fairlearn/widget/fairlearnDashboard.py

```diff
@@ -126,20 +121,34 @@ def __init__(
                                   if "classification" in method[1]["model_type"]]
         regression_methods = [method[0] for method in self._metric_methods.items()
                               if "regression" in method[1]["model_type"]]
+        
+        dataset = self._sanitizeDataShape(sensitive_features)
+        self._predicted_ys = self._convertToList(predicted_ys)
+        if len(np.shape(self._predicted_ys)) == 1:
+            self._predicted_ys = [self._predicted_ys]
+        self._true_y = self._convertToList(true_y)
+
+        if np.shape(self._true_y)[0] != np.shape(self._predicted_ys)[1]:
+            raise ValueError("Predicted Y does not match true y shape")
+        
+        if np.shape(self._true_y)[0] != np.shape(dataset)[0]:
+            raise ValueError("Sensitive features shape does not match true y shape")
+
 
         dataArg = {
-            "true_y": self._convertToList(true_y),
-            "predicted_ys": self._convertToList(predicted_ys),
-            "dataset": self._convertToList(sensitive_features),
+            "true_y": self._true_y,
```

Comment:
why do we call it true_y, but predicted_ys with s at the end?

Reply:
oh because it's literally multiple ... got it

Reply:
this is a good terminology question, true y is 1d, where predicted ys is 2d, one per model. If a 1d value is passed in for predicted ys, we will wrap it to make it 2d. If the terminology is bad, I'll change the names